bumble/avdtp.py

```diff
@@ -51,6 +51,7 @@
     A2DP_MPEG_2_4_AAC_CODEC_TYPE,
     A2DP_NON_A2DP_CODEC_TYPE,
     A2DP_SBC_CODEC_TYPE,
+    A2DP_VENDOR_MEDIA_CODEC_INFORMATION_CLASSES,
```

Comment:
(Just a comment)

From a spec perspective, importing A2DP from AVDTP seems like a reversed dependencies, but top-down parsing is important for usage. I am thinking whether it's possible to hook parsing table from A2DP, but that will require callers to always import A2DP, or we will need to wrap AVDTP and A2DP stubs inside a public module which is too aggresive for now.

Reply:
Yes, it would be nice to have some better way to have a sort of dependency injection here. I haven't found a great way to do that that's not intrusive. I'll try to think of some better way.